src/main/java/seedu/address/storage/JsonSerializableAddressBook.java

```diff
@@ -68,6 +74,13 @@ public AddressBook toModelType() throws IllegalValueException {
             }
             addressBook.addPerson(person);
         }
+        for (JsonAdaptedVendor jsonAdaptedVendor : vendors) {
```

Comment:
Not sure how feasible it is but maybe we can consider ensuring there is no double entry? Maybe can check if the Person is already saved as JsonAdaptedPerson? Or save the vendor first den when saving People, if it already exists as a Vendor do not save as a Person?

Idt this is a critcal bug so can always fix it next iteration also :)

Reply:
Hmm actually the idea is that i wanted duplication, since a vendor is a person also, i was thinking it should appear in both? because in the `vendorlist` and `personlist`, if a person is also a vendor, the person is stored in both lists. so that elsewhere in the code if there is a need to iterate through the personlist, can just do that instead of having to iterate through both lists. only problem is in the edit command, if the person is edited the vendor needs to be updated